apps/nextjs-website/react-components/types/Accordion/Accordion.types.ts

```diff
@@ -11,5 +11,6 @@ export interface AccordionProps extends SectionProps {
   subtitle?: string;
   description?: string | JSX.Element;
   accordionItems: AccordionItemProps[];
-  layout?: 'left' | 'center' | 'right';
-}
\ No newline at end of file
+  layout?: 'left' | 'center';
+  textAlignment?: 'left' | 'center' | 'right';
```

Comment:
```suggestion
  layout: 'left' | 'center';
  textAlignment: 'left' | 'center' | 'right';
```
Make the above fields required.